### DISCUSSION TITLE
Embedded-cassandra >4.0.1 with JDK11

### DISCUSSION THREAD

**Discussion by giom-l** [nosan/embedded-cassandra#372] [Q&A]
Hello, 

I read multiple issues mentioning that Cassandra prior to 4.0 were not supporting JDK11. 
However, I'm wondering why versions of embedded-cassandra >4.0.1 (which itself embeds cassandra-driver > 4.5 [that is fully supporting Cassandra 4.X](https://docs.datastax.com/en/driver-matrix/docs/java-drivers.html) are still built with Java 1.8 ? 

Is it planned to upgrade to java 11 sooner or later ? 
Or maybe I'm missing something :)

**Comment by nosan** [comment]
Hello, 

You can use any cassandra-drivers (old, new, any versions)

The code of embedded-cassandra is written using JDK 1.8


embedded-cassandra has only two compile dependencies:

```
<dependecies>
    <dependency>
        <groupId>org.apache.commons</groupId>
        <artifactId>commons-compress</artifactId>
        <version>1.24.0</version>
    </dependency>
    <dependency>
        <groupId>org.yaml</groupId>
        <artifactId>snakeyaml</artifactId>
        <version>2.2</version>
    </dependency>
</dependecies>
```
